### ISSUE TITLE
rewrite file refactoring

### ISSUE THREAD

**Issue by nittka** [thSoft/elysium#134]
When uri resolution and include validation have been reviewed, the corresponding changes have to be merged into the file refactoring code.

**Comment by nittka** [comment]
When working in this issue, make sure that only neccessary checks are executed - only once (see #136 checks when deleting non-related files and folders).

**Comment by thSoft** [comment]
I schedule this for the next release because at the moment LilyPond files cannot be deleted in Elysium.

**Comment by thSoft** [comment]
Sorry, as of a3b081d3788452bb77cd32a26d287f885b47ab12, files can be deleted, so there is no regression. However, these changes are related closely to URI resolution, so it would be great

**Comment by nittka** [comment]
Yes, it's part of the include refactoring package. I'll look into it soon. But after the first big refactoring, I first wanted to do deal with some smaller issues.

**Comment by nittka** [comment]
Dealt with in #174.
